gettsim/social_insurance.py

```diff
@@ -26,185 +24,193 @@ def soc_ins_contrib(person, tb):
-    pauschmini = tb["mini_ag_gkv"] + tb["mini_ag_grv"] + tb["stpag"]
+    pauschmini = params["mini_ag_gkv"] + params["mini_ag_grv"] + params["stpag"]
     f = round(pauschmini / dbsv, 4)
     return f
 
 
-def calc_midi_bemessungsentgelt(person, tb):
-    f = calc_midi_f(tb)
-    return f * tb["mini_grenzew"] + (
-        (tb["midi_grenze"] / (tb["midi_grenze"] - tb["mini_grenzew"]))
-        - (tb["mini_grenzew"] / (tb["midi_grenze"] - tb["mini_grenzew"]) * f)
-    ) * (person["m_wage"] - tb["mini_grenzew"])
+def calc_midi_bemessungsentgelt(person, params):
+    f = calc_midi_f(params)
+    return f * params["mini_grenzew"] + (
+        (params["midi_grenze"] / (params["midi_grenze"] - params["mini_grenzew"]))
+        - (
+            params["mini_grenzew"]
+            / (params["midi_grenze"] - params["mini_grenzew"])
+            * f
+        )
+    ) * (person["m_wage"] - params["mini_grenzew"])
 
 
-def calc_midi_old_age_pensions_contr(person, tb):
+def calc_midi_old_age_pensions_contr(person, params):
     """ Calculate old age pensions social insurance contribution for midi job."""
-    grbetr_rv = 2 * tb["grvbs"] * person["bemessungsentgelt"]
-    ag_rvbeit = tb["grvbs"] * person["m_wage"]
+    grbetr_rv = 2 * params["grvbs"] * person["bemessungsentgelt"]
+    ag_rvbeit = params["grvbs"] * person["m_wage"]
     return grbetr_rv - ag_rvbeit
 
 
-def calc_midi_health_contr(person, tb):
+def calc_midi_health_contr(person, params):
     """ Calculate social insurance health contributions for midi job."""
-    grbetr_gkv = (tb["gkvbs_an"] + tb["gkvbs_ag"]) * person["bemessungsentgelt"]
-    ag_gkvbeit = tb["gkvbs_ag"] * person["m_wage"]
+    grbetr_gkv = (params["gkvbs_an"] + params["gkvbs_ag"]) * person["bemessungsentgelt"]
+    ag_gkvbeit = params["gkvbs_ag"] * person["m_wage"]
     return grbetr_gkv - ag_gkvbeit
 
 
-def calc_midi_unemployment_contr(person, tb):
-    grbetr_alv = 2 * tb["alvbs"] * person["bemessungsentgelt"]
-    ag_avbeit = tb["alvbs"] * person["m_wage"]
+def calc_midi_unemployment_contr(person, params):
+    grbetr_alv = 2 * params["alvbs"] * person["bemessungsentgelt"]
+    ag_avbeit = params["alvbs"] * person["m_wage"]
     return grbetr_alv - ag_avbeit
 
 
-def calc_midi_long_term_care_contr(person, tb):
-    grbetr_pv = 2 * tb["gpvbs"] * person["bemessungsentgelt"]
-    ag_pvbeit = tb["gpvbs"] * person["m_wage"]
+def calc_midi_long_term_care_contr(person, params):
+    grbetr_pv = 2 * params["gpvbs"] * person["bemessungsentgelt"]
+    ag_pvbeit = params["gpvbs"] * person["m_wage"]
     if ~person["haskids"] & (person["age"] > 22):
-        return grbetr_pv - ag_pvbeit + tb["gpvbs_kind"] * person["bemessungsentgelt"]
+        return (
+            grbetr_pv - ag_pvbeit + params["gpvbs_kind"] * person["bemessungsentgelt"]
+        )
     else:
         return grbetr_pv - ag_pvbeit
 
 
-def no_midi(person, tb):
+def no_midi(person, params):
     """Dummy function returning nothing
     """
     person[OUT_COLS] = 0.0
```

Comment:
Do not rely on globals. Why not use `pass` ?

Reply:
The global is defined in the beginning of the module and is just the five social security contributions.

Reply:
Then at least fix the docstring.

Reply:
Reformulated docstring!